packages/trader/src/Stores/Modules/Trading/Helpers/proposal.ts

```diff
@@ -26,28 +53,40 @@ export const getProposalErrorField = response => {
     return el_error && isVisible(el_error) ? error_id : null;
 };
 
-export const getProposalInfo = (store, response, obj_prev_contract_basis) => {
-    const proposal = response.proposal || {};
-    const profit = proposal.payout - proposal.ask_price || 0;
+export const getProposalInfo = (
+    store: TTradeStore,
+    response: PriceProposalResponse & TError,
+    obj_prev_contract_basis: TObjContractBasis
+) => {
+    const proposal = response.proposal || ({} as Proposal);
+    const profit = (proposal.payout || 0) - (proposal.ask_price || 0);
     const returns = (profit * 100) / (proposal.ask_price || 1);
     const stake = proposal.display_value;
     const basis_list = store.basis_list;
 
-    const contract_basis =
+    const contract_basis: TObjContractBasis | undefined =
         store.is_vanilla || store.is_turbos
-            ? { text: getLocalizedBasis().payout_per_point, value: 'display_number_of_contracts' }
-            : basis_list.find(o => o.value !== store.basis) || {};
+            ? {
+                  text: store.is_vanilla ? getLocalizedBasis().payout : getLocalizedBasis().payout_per_point,
```

Comment:
@kate-deriv let's keep `getLocalizedBasis().payout_per_point` from master for both vanillas and turbos  :)

Reply:
Will do, thank you